src/apps/sources/pages/EditSourcePage.tsx

```diff
@@ -3,6 +3,7 @@ import {Grid, Paper} from "@material-ui/core";
 import {connect} from "react-redux";
 import {Redirect, useLocation} from "react-router-dom";
 import {
+    editSourceAction,
```

Comment:
This needs to match the names of the functions exported in `/sources/redux/actions.ts`